kernel/src/main/java/org/apache/shardingsphere/elasticjob/kernel/internal/schedule/JobScheduler.java

```diff
@@ -80,7 +82,15 @@ public JobScheduler(final CoordinatorRegistryCenter regCenter, final ElasticJob
         Collection<ElasticJobListener> jobListeners = getElasticJobListeners(this.jobConfig);
         setUpFacade = new SetUpFacade(regCenter, this.jobConfig.getJobName(), jobListeners);
         schedulerFacade = new SchedulerFacade(regCenter, this.jobConfig.getJobName());
-        jobFacade = new JobFacade(regCenter, this.jobConfig.getJobName(), jobListeners, findTracingConfiguration().orElse(null));
+        
+        // the single sharding scenario and specified SINGLE_SHARDING_BALANCE strategy
```

Comment:
Please move the comment to the method signature, or remove the comment. If you put the comment inside the method body, the compiled JavaDoc may not see it.

Reply:
done